example/resources/js/app.js

```diff
@@ -12,17 +12,25 @@ document.addEventListener('block-editor/init', () => {
     console.log('block-editor/init')
 })
 
-const mediaUpload = ({filesList, onFileChange}) => {
-    setTimeout(() => {
-        const uploadedFiles = Array.from(filesList).map(file => {
-            return {
-                id: file.name,
-                name: file.name,
-                url: `https://dummyimage.com/600x400/000/fff&text=${file.name}`
-            }
-        })
-        onFileChange(uploadedFiles)
-    }, 1000)
+const mediaUpload = async ({ filesList, onFileChange }) => {
```

Comment:
Great that you added this! Will definitely help people understand how it works :)

Just wanted to ask if you can make sure the code style in this file is the same (with `;` or without, for example).